data/racecar/block.urdf

```diff
@@ -0,0 +1,19 @@
+<robot name="magnet_block">
+  <link name="magnet_block">
+    <collision>
+      <origin xyz="0 0 0.029" />
```

Comment:
I think this offset should be set in the simulation instead of here to avoid confusion.